hm_1.py

```diff
@@ -0,0 +1,5 @@
+num = input('Num: ')
```

Comment:
Please add a task description as a docstring into module with task implementation instead of Pull request description.

It should be clear what is it about reading a module.

Reply:
Added docstring before the module